modules/smart-agent_elasticsearch/conf/01-cluster-status.yaml

```diff
@@ -0,0 +1,19 @@
+module: ElasticSearch
+name: "cluster status"
+aggregation: true
```

Comment:
If you have multiple clusters, this will do the transformation on all clusters :smiley: